lib/y2_module_basetest.pm

```diff
@@ -31,8 +31,25 @@ our @EXPORT = qw(is_network_manager_default
   continue_info_network_manager_default
   accept_warning_network_manager_default
   workaround_suppress_lvm_warnings
+  with_yast_env_variables
 );
 
+=head2 with_yast_env_variables
+
+ with_yast_env_variables([extra_vars]);
+
+Set environment variables for yast application.
+C<extra_vars> extends the variables that we can use. C<extra_vars> expects a string.
+ex: with_yast_env_variables("foo=bar");
+
+=cut
+sub with_yast_env_variables {
+    my ($self, $extra_vars) = @_;
+    my $vars = "Y2DEBUG=1 ZYPP_MEDIA_CURL_DEBUG=1 Y2STRICTTEXTDOMAIN=1";
+    $vars .= " $extra_vars" if ($extra_vars);
+    return $vars;
```

Comment:
As you are not using `$self`, you can possibly shift oradd a default value.
```perl
my $extra_vars = shift // '' ;
return  "Y2DEBUG=1 ZYPP_MEDIA_CURL_DEBUG=1 Y2STRICTTEXTDOMAIN=1 $extra_vars";
```

So you can have `$extra_vars` as defined in case you are calling the function in void context.

Reply:
If you need to pass the $self, but there is not a need to use in the body, I would personally prefer to use `undef` instead of saving something in `$self`. But that is a matter of taste rather than any improvement